Tella/Scenes/GDrive/Views/GdriveReportMainView.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  GdriveReportMainView.swift
+//  Tella
+//
+//  Created by RIMA on 6/9/2024.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+struct GdriveReportMainView: View {
+    
+    @ObservedObject var reportsMainViewModel: GDriveViewModel
+    
+    var body: some View {
+        ReportMainView(reportsMainViewModel: reportsMainViewModel) { id in
```

Comment:
Don't forget to implement the changes we talked about [here](https://github.com/Horizontal-org/Tella-iOS/pull/183/files/c65f4bef0799d23399e93f69ed5ed4c0131ed412#diff-08c42b92e2cde496f02dfa7c08f7d1477457bc4de31b0c119cbd85b7bfc1ef79). Probably you are working on this but I'll add this comment just to not forget it